src/components/playground/PlaygroundComponent/PlaygroundComponent.module.scss

```diff
@@ -90,7 +90,7 @@
       border-radius: 4px;
       border: solid 2px var(--gray-04);
       color: var(--black-03);
-      background-color: var(--white-01);
+      background-color: var(--white);
```

Comment:
I'm not sure how to feel about this. Since there is only one white, why not use just `white` ?

Reply:
I would still suggest to keep it, there can be slight tint variations of white. For example a white with some yellow/orange mixed in to make the background softer on the eyes. Perhaps I should just change the name of the variable instead to something else (maybe --paper or something like that). I think it is still beneficial to have a  global color that can be adjusted easily.